pkg/settings/features.go

```diff
@@ -16,13 +19,18 @@ const (
 	Ride4DApps // RIDE V3
 	OrderV3
 	ReduceNFTFee
-	BlockReward      // 14
-	BlockV5          // 15
-	RideV5           // 16
-	RideV6           // 17
-	InvokeExpression // 18
+	BlockReward           // 14
+	BlockV5               // 15
+	RideV5                // 16
+	RideV6                // 17
+	ConsensusImprovements // 18
+	InvokeExpression      // 19
 )
 
+func (f Feature) Description() string {
```

Comment:
Why do we need `Description`. No usages again.

Reply:
`Implemented` would be useful. But description only used in API in iteration over all features.
I think such functions should be introduced along with code that use them.

Reply:
Agree, fixed